### PR TITLE
Make the flashbots usage optional on the runtime dockerfile

### PR DESCRIPTION
**Pull request by angrybayblade**
## Proposed changes

Make the usage of the flashbots plugin optional on the runtime image and add HWI plugin as a dependency on the base image

## Fixes

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: deployments/Dockerfiles/autonomy/requirements.txt [hunk 1]
```diff
@@ -3,5 +3,6 @@ requests==2.27.1
 open-aea[all]==1.31.0
 open-aea-cli-ipfs==1.31.0
 open-aea-ledger-ethereum==1.31.0
+open-aea-ledger-ethereum-hwi==1.31.0
```
[the remark this answers is not in the document]
Comment (dagacha) [reply deployments/Dockerfiles/autonomy/requirements.txt:6]: Will that image be part of 0.11.0?

[the remark this answers is not in the document]
Comment (dagacha) [reply deployments/Dockerfiles/autonomy/requirements.txt:6]: Let's make sure it is added to the v0.11.0 features list

**Comment by dagacha** [comment]
LGTM - I believe @DavidMinarsch wanted to review

File: autonomy/data/Dockerfiles/agent/start.sh [hunk 2]
```diff
@@ -34,15 +39,27 @@ then
...
     aea generate-key cosmos --connection
     aea add-key cosmos --connection || (echo "Failed to generate the cosmos key needed for libp2p connection" && exit 1)
     aea add-key ethereum
-    aea add-key ethereum-flashbots
+    
+    if grep "open-aea-ledger-ethereum-flashbots" aea-config.yaml -q
+    then
+        aea add-key ethereum_flashbots
+        cp ethereum_private_key.txt ethereum_flashbots_private_key.txt
+    fi
```
Comment (Adamantios) [inline autonomy/data/Dockerfiles/agent/start.sh:61]: Same

**Review by 0xArdi — APPROVED** [review]

**Review by Adamantios — APPROVED** [review]

**Pull request merged**